x/stakeibc/keeper/ibc_handlers.go

```diff
@@ -164,7 +164,11 @@ func (k *Keeper) HandleSend(ctx sdk.Context, msg sdk.Msg, sequence string) error
 				k.Logger(ctx).Error("epochUnbondingRecord.UnbondingEpochNumber == dayEpochTracker.EpochNumber")
 				continue
 			}
-			hostZoneUnbonding := epochUnbondingRecord.HostZoneUnbondings[zone.ChainId]
+			hostZoneUnbonding, found := k.RecordsKeeper.GetHostZoneUnbondingByChainId(ctx, epochUnbondingRecord.EpochNumber, zone.ChainId)
```

Comment:
just making a note so we don't forget - this has moved into the callbacks so we'll want to make sure to port it over when resolving conflicts

Reply:
good callout!